keep-ui/app/alerts/alert-note-modal.tsx

```diff
@@ -0,0 +1,121 @@
+'use client';
+
+import React, { useContext, useState } from 'react';
+const ReactQuill = typeof window === 'object' ? require('react-quill') : () => false;
+import 'react-quill/dist/quill.snow.css';
+import { Button } from '@tremor/react';
+import { getApiURL } from '../../utils/apiUrl';
+import { useSession } from 'next-auth/react';
+
+interface AlertNoteModalProps {
+  isOpen: boolean;
+  handleClose: () => void;
+  initialContent: string;
+  alertFingerprint: string;
+}
+
+const AlertNoteModal: React.FC<AlertNoteModalProps> = ({
```

Comment:
Using React.FC isn't recommended anymore 😢

Reply:
you are right